### PR TITLE
Fix comment for method SetForTests

### PR DESCRIPTION
**Pull request by yu-song**
cc @dchen1107

### CODE REVIEW

**Comment by yu-song** [comment]
I signed it!

**Comment by resouer** [comment]
@k8s-bot ok to test

File: pkg/capabilities/capabilities.go [hunk 1]
```diff
@@ -68,7 +68,7 @@ func Setup(allowPrivileged bool, privilegedSources PrivilegedSources, perConnect
 	})
 }
 
-// SetCapabilitiesForTests.  Convenience method for testing.  This should only be called from tests.
+// SetForTests set capabilities for tests.  Convenience method for testing.  This should only be called from tests.
```
Comment (resouer) [inline pkg/capabilities/capabilities.go]: nit: `SetForTests sets ...`

**Review by resouer — COMMENTED** [review]
One nit, others lgtm

**Comment by yu-song** [comment]
@resouer   I have modified it.

**Comment by dchen1107** [comment]
/lgtm 
and
/approve

Thanks!

**Comment by yu-song** [comment]
@dchen1107 
Excuse me, because there is no lgtm label, one check hasn’t completed yet, can you help make a label again?
Thanks very much.

**Comment by yu-song** [comment]
test again

**Pull request merged**